de.fabrik19.mos.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: de.fabrik19.mos.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mos-launcher
+modules:
+  - name: mos-launcher
+    buildsystem: simple
+    build-options:
+      strip: false
+      no-debuginfo: true
+    build-commands:
```

Comment:
you should use `install` instead of `cp`, it create the missing directories and can set the modes.

Reply:
Used `install` instead of `cp`.